handlers/meal.ts

```diff
@@ -8,14 +8,14 @@ class MealPlanHandle implements IMealPlan {
     groupId: string = "";
 
     constructor(private gun: Gun) {
-            this.user = gun.user(userPub);
-            this.user.get("group").get("groupId").on((data: string) => {
-                this.groupId = data.toString()     
+            this.user = this.gun.user(userPub);
+            this.user.get("groupId").once((data: any) => {
```

Comment:
data: string, also are you sure we dont want on here? Will it not fail if I leave a group and join another

Reply:
GroupId bliver lavet med .put så der vil altid kun være et resultat. Derudover er der ikke rigtigt implementeret at forlade/joine andre grupper så ved ikke hvor meget funktionalitet skulle omskrives for at det skulle virke.